Src/MoneyFox.Ui/Common/Groups/DateListGroupCollection.cs

```diff
@@ -73,9 +76,10 @@ private DateListGroupCollection(string key, RelayCommand<T>? itemClickCommand =
         foreach (var item in items)
         {
             var index = getKey(item);
+            var infuture = getSortKey(item).Date > DateTime.Today;
```

Comment:
I wonder if it would make sense to move that to the converter and then pass the date of the payment item instead of handling this here. What do you mean?

Reply:
@NPadrutt - Great catch. I just implemented your suggestion and submitted a new PR.